### DISCUSSION TITLE
Heatmap in Tradingview chart

### DISCUSSION THREAD

**Discussion by 1ik** [tradingview/lightweight-charts#1078] [Q&A] [answered]
We are trying to build a chart like this https://www.tradingview.com/heatmap/stock/?color=change&dataset=SPX500&group=sector&size=market_cap_basic
with the trading view library on our own stock data. But can't seem to find any javascript api for it. Please help.

Thanks

**Comment by romfrancois** [comment, marked as answer]
Hi @1ik !

Im afraid it's a feature only available on our main website.
